packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -2204,7 +2204,7 @@ describeCompat(
 			},
 		);
 
-		itExpects(
+		itExpects.skip(
```

Comment:
Please let it run for localServer, but you can skip for other drivers while I investigate.  This is a new failure compared to the one I fixed last time.